### PR TITLE
Fix #177574 ai.meta.com

### LINKED ISSUE #177574: ai.meta.com
### Issue URL (Social Widget)
 [https://ai.meta.com/blog/meta-llama-3/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fai.meta.com%2Fblog%2Fmeta-llama-3%2F)
Username: `@rakleed`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Windows v7.17.0 (4705)
Browser: | Chrome
License type: | paid
AdGuard driver: | WFP
Stealth mode options: | Strip URLs from tracking parameters, <br>Hide your search queries, <br>Hide your Referrer from third-parties (ai.meta.com), <br>Hide your User-Agent, <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Self-destructing third-party cookies (180), <br>Disable cache for third-party requests, <br>Block trackers, <br>Disable WAP Push Message Routing Service, <br>Disable Windows Defender automatic samples submission, <br>Disable Windows Telemetry, <br>Turn off Advertising ID
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets<br/><br/><b>Language-specific:</b><br/>AdGuard Russian
Userscripts: | https://update.greasyfork.org/scripts/433360/Nova%20YouTube.user.js<br>https://update.greasyfork.org/scripts/15222/IMDb%20Tomatoes.user.js<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by ksaduakasov**
Fix #177574

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -75,6 +75,7 @@ mentalfloss.com,90min.com,fansided.com,90min.de,12thmanrising.com,1428elm.com,8p
...
 !
+ai.meta.com##._amgj > div._9bhp:has(> div._9bho)
```
Comment (piquark6046) [inline SocialFilter/sections/specific.txt:78]: I prefer `div[class^="_"] > div[class*=" _"]:has(> div[class^="_"] > a[href*="facebook.com/sharer/sharer.php"])`.

File: AnnoyancesFilter/Popups/sections/subscriptions_specific.txt [hunk 2]
```diff
@@ -54,6 +54,7 @@ mentalfloss.com,90min.com,fansided.com,90min.de,12thmanrising.com,1428elm.com,8p
 !
 ! SECTION: Subscriptions - Regular rules
 !
+ai.meta.com##._amgj > div._9bhp:has(div._a5gf > div._amh_)
```
Comment (piquark6046) [inline AnnoyancesFilter/Popups/sections/subscriptions_specific.txt:57]: I prefer `div[class*=" _"] > div[class*=" _"]:has(> div[style] a[href$="/subscribe/"])`.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
please rename title of the PR to `Fix #177574 ai.meta.com`.

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:78]: without space in the second div maybe?
`div[class^="_"] > div[class*="_"]:has(> div[class^="_"] > a[href*="facebook.com/sharer/sharer.php"])`

**Review by piquark6046 — APPROVED** [review]

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request merged**